linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_default_forwarding/rule.yml

```diff
@@ -0,0 +1,35 @@
+documentation_complete: true
+
+prodtype: sle15
+
+title: 'Disable Kernel Parameter for IPv6 Forwarding by default'
+
+description: '{{{ describe_sysctl_option_value(sysctl="net.ipv6.conf.default.forwarding", value="0") }}}'
+
+rationale: |-
+    IP forwarding permits the kernel to forward packets from one network
+    interface to another. The ability to forward packets between two networks is
+    only appropriate for systems acting as routers.
+
+severity: medium
+
+identifiers:
+    cce@sle15: CCE-85716-9
+
+references:
+    disa@sle15: CCI-000366
+    srg@sle15: SRG-OS-000480-GPOS-00227
+    nist@sle15: CM-6(b),CM-6.1(iv)
+    stigid@sle15: SLES-15-040382
+
```

Comment:
Please add also ocil_clause. See other similar rules for examples. ocil_clause could be here something like "The sysctl option is not correctly configured"

Reply:
Does this look right?

Reply:
Yes, thank you.